packages/vue/src/components/atoms/SfIcon/SfIcon.vue

```diff
@@ -91,9 +91,10 @@ export default {
       return SF_ICONS.includes(this.icon.trim());
     },
     iconViewBox() {
-      return this.isSFIcons
-        ? icons[this.icon].viewBox || this.viewBox
+      const defaultSFIconViewBox = this.isSFIcons
+        ? icons[this.icon].viewBox
         : this.viewBox;
+      return this.viewBox || defaultSFIconViewBox;
```

Comment:
@aniamusial it's doesn't work :) because this.viewBox is always true, you set a default to '0 0 24 24'